docs/reference/pip_install.rst

```diff
@@ -82,20 +82,12 @@ Requirements File Format
 Each line of the requirements file indicates something to be installed,
 and like arguments to :ref:`pip install`, the following forms are supported::
 
-    <requirement specifier>
     <requirement specifier> [--install-option="..."] [--global-option="..."]
```

Comment:
We should add the ; marker to the example line to make the actual form we support clear.